rllib/utils/minibatch_utils.py

```diff
@@ -113,7 +113,7 @@ def get_len(b):
             # Note (Kourosh): env_steps is the total number of env_steps that this
             # multi-agent batch is covering. It should be simply inherited from the
             # original multi-agent batch.
-            minibatch = MultiAgentBatch(minibatch, len(self._batch))
+            minibatch = MultiAgentBatch(minibatch, self._minibatch_size)
```

Comment:
I am not sure if I understand whaat is the diff? What was wrong with len(self._batch) that this would solve?

Reply:
`self._batch` is the entire MultiAgentBatch, so it's not a good base to compute the size of the sliced sub-batches.
`len(minibatch)` is also not good here as minibatch is a dict. :|

Reply:
can you add this as a comment like the Note above?

Reply:
done